gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/mapper/ApiMessageLogsMapper.java

```
@@ -38,13 +41,100 @@ public interface ApiMessageLogsMapper {
     @Mapping(target = "connectorType", expression = "java(mapConnectorType(messageMetrics.getConnectorType()))")
     ApiMessageLog map(MessageLog messageMetrics);
 
+    @Mapping(target = "additional", expression = "java(parseAdditionalParams(searchMessageMetricsParam.getAdditional()))")
     SearchMessageLogsFilters map(SearchMessageLogsParam searchMessageMetricsParam);
 
     default MessageOperation mapOperation(String operation) {
-        return Optional.ofNullable(operation).map(MessageOperation::fromValue).orElse(null);
+        if (operation == null) {
```

That's a bad idea, we should not hide errors, if this conversion does not work that means that there an operation the API does not know about... so API tests should failed rather than silently log something, tbh, noone will ever read.

---

Understood. Addressed in [665dec0](https://github.com/gravitee-io/gravitee-api-management/pull/14303/commits/665dec04a553bc8495231deadc3a75d1b4e45707).